src/estimation/gsa.jl

```diff
@@ -3,23 +3,46 @@ function DiffEqSensitivity.gsa(m::PumasModel,subject::Subject,params::NamedTuple
     vhighparam = TransformVariables.inverse(toidentitytransform(m.param),p_range_high)
     p_range = [[vlowparam[i],vhighparam[i]] for i in 1:length(vlowparam)]
     trf_ident = toidentitytransform(m.param)
+    sim_ = simobs(m,subject,params,args...;kwargs...)
+    length_vars = append!([0],cumsum([length(getproperty(sim_.observed,key)) for key in vars]))
```

Comment:
```suggestion
    length_vars = append!([0], cumsum([map(length, sim_.observed)...]))
```
Generally, you should avoid using `getproperty` when accessing `NamedTuple`s. Also, please add spaces after commas and around most operators to make the code more readable.